tests/backup/backup_test.go

```diff
@@ -6296,11 +6298,20 @@ var _ = Describe("{DeleteAllBackupObjects}", func() {
 			ctx, err := backup.GetAdminCtxFromSecret()
 			dash.VerifyFatal(err, nil, "Getting context")
 			preRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, preRuleNameList[0])
```

Comment:
Add the if condition for pre rule as well

Reply:
Done